src/mlpack/tests/feedforward_network_test.cpp

```diff
@@ -410,4 +410,14 @@ BOOST_AUTO_TEST_CASE(DropConnectNetworkTest)
       (dataset, labels, dataset, labels, 2, 10, 50, 0.2);
 }
 
+BOOST_AUTO_TEST_CASE(NamedNetworkTest)
+{
+  FFN<NegativeLogLikelihood<> > model;
+  model.AddNamed<LinearNoBias<> >("input", 4, 10);
+  model.Add<SigmoidLayer<> >();
+  model.Add<LinearNoBias<> >(10, 2);
+  model.Add<LogSoftMax<> >();
+  std::cout << boost::apply_visitor(WeightSizeVisitor(), model.getByName("input") ) << std::endl;
```

Comment:
This test doesn't actually test anything.  What are you trying to test?  You should be using the `BOOST_REQUIRE_EQUAL()` or similar tests to ensure that the functionality is correct here.